src/components/NavBar/NavBar.js

```diff
@@ -1,53 +1,108 @@
 import { ReactComponent as ThemeIcon } from "../SVG/themeIcon.svg";
-import { ReactComponent as CurrencySymbol } from "../SVG/currencySymbol.svg";
-import { ReactComponent as DownArrow } from "../SVG/downArrow.svg";
 import { ReactComponent as SearchIcon } from "../SVG/searchIcon.svg";
 
 class NavBar extends React.Component {
+  state = {
+    clickedLink: false,
+    currency: JSON.parse(localStorage.getItem("currency")),
+    showDropdown: false,
+  };
+  handleDropdown = () => {
+    this.setState({ showDropdown: !this.state.showDropdown });
+  };
+  handleDropdownFocus = () => {
+    this.setState({ showDropdown: false });
+  };
+  handleSelectedCurrency = currency => {
+    this.setState({
+      currency: currency,
+      showDropdown: !this.state.showDropdown,
+    });
+    this.props.handleSelectedCurrency(currency);
+  };
+  handleClickedLink = () => {
+    this.setState({ clickedLink: !this.state.clickedLink });
+  };
   render() {
     return (
       <Nav>
         <Ul>
           <Container>
             <LeftContainer>
-              <Li>
-                <StyledLink to="/"> Coins </StyledLink>
-              </Li>
-              <Li>
-                <StyledLink to="/Portfolio"> Portfolio </StyledLink>
-              </Li>
+              <LinkLi>
+                <StyledLink to="/">
+                  <LinkWrapper
+                    onClick={this.handleClickedLink}
+                    currentPage={window.location.pathname === "/"}
+                  >
+                    Coins
+                  </LinkWrapper>
+                </StyledLink>
+              </LinkLi>
+              <LinkLi>
+                <StyledLink to="/Portfolio">
+                  <LinkWrapper
+                    onClick={this.handleClickedLink}
+                    currentPage={window.location.pathname === "/Portfolio"}
+                  >
+                    Portfolio
+                  </LinkWrapper>
+                </StyledLink>
+              </LinkLi>
             </LeftContainer>
             <RightContainer>
               <Li>
                 <InputContainer>
-                  <SearchIcon fill= {this.props.theme.color}/>
+                  <SearchIcon fill={this.props.theme.color} />
                   <Input type="text" placeholder="Search..." />
                 </InputContainer>
               </Li>
+
               <Li>
-                <CurrencyContainer>
-                  <CurrencySymbol />
-                  <CurrencyContainer>
-                    <div>USD</div>
-                    <DownArrow />
+                <Wrapper>
+                  <CurrencyContainer onClick={this.handleDropdown}>
+                    <CurrencySymbol>
+                      {this.state.currency === "USD"
+                        ? "$"
+                        : this.state.currency === "EUR"
+                        ? "€"
+                        : this.state.currency === "GBP"
+                        ? "£"
+                        : ""}
```

Comment:
maybe use one universal object where you can grab these values here and above where i commented

Reply:
Placed currencies into an array and mapped through them